DeepLinkSDK/RouteHandler/DPLRouteHandler.h

```diff
@@ -51,6 +58,10 @@
  */
 - (UIViewController *)viewControllerForPresentingDeepLink:(DPLDeepLink *)deepLink;
 
+/**
+ returns the first `UINavigationController'
+ */
+- (UINavigationController *)navigationControllerForPresentingDeepLink:(DPLDeepLink *)deepLink;
```

Comment:
In the spirit of keeping the library generic. I agree with your suggestions, and I think subclassing `DPLRouteHandler` and overriding `viewControllerForPresentingDeepLink` does it. 

Thanks @wessmith